packages/react/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -61,18 +61,25 @@ export const SearchBoxBase: React.FunctionComponent<ISearchBoxProps> = React.for
   } = props;
 
   const [hasFocus, setHasFocus] = React.useState(false);
+
+  const prevChangeTimestamp = React.useRef<number | undefined>();
   const [uncastValue, setValue] = useControllableValue(
     props.value,
     defaultValue,
-    React.useCallback(
-      (ev: React.ChangeEvent<HTMLInputElement> | undefined) => {
-        onChange?.(ev, ev?.target.value);
-        onChanged?.(ev?.target.value);
-      },
-      [onChange, onChanged],
-    ),
+    (ev: React.ChangeEvent<HTMLInputElement> | undefined, newValue: string) => {
```

Comment:
Do we no longer need `React.useCallback` here?

Reply:
In this case it's not needed because `useControllableValue` always uses the most recently passed callback.